demo/src/main/java/com/novoda/merlin/demo/connectivity/display/MerlinSnackbar.java

```diff
@@ -0,0 +1,49 @@
+package com.novoda.merlin.demo.connectivity.display;
+
+import android.content.res.Resources;
+import android.support.annotation.StringRes;
+import android.support.design.widget.Snackbar;
+import android.view.View;
+
+import com.novoda.merlin.demo.R;
+
+class MerlinSnackbar {
+
+    private static final String EMPTY_MESSAGE = "";
+    private final Snackbar snackbar;
+
+    static MerlinSnackbar withDuration(Resources resources, View attachTo) {
```

Comment:
method name is a bit confusing/misleading given we're not passing in a duration of any sorts